codespell_lib/data/dictionary.txt

```diff
@@ -6899,9 +6907,11 @@ lenths->lengths
 leran->learn
 lerans->learns
 lesstiff->lesstif
+lest->least
```

Comment:
`lest` is a valid word. It's old english but nevertheless still used. On the fence about adding it as a result though.

Reply:
I use `lest` in code sometimes, so we should keep it as an option

Reply:
Then I propose `lest->lest, least,`

Reply:
Removed.